File: C++/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P09_Volleyball.cpp
#include <iostream>
#include <string>
#include <cmath>
using namespace std;

int main() {

    string year;
    int holidays, weekendsHome;
    cin >> year >> holidays >> weekendsHome;

    int weekendsPerYear = 48;

    double playHolidays = holidays * (2.0 / 3.0);
    double playWeekendsSofia = (weekendsPerYear - weekendsHome) * (3.0 / 4.0);
    double playInTotal = floor(playWeekendsSofia + weekendsHome + playHolidays);
    double playInTotalLeap = floor((playWeekendsSofia + weekendsHome + playHolidays) * 1.15);

    if (year == "leap") {
        cout << playInTotalLeap << endl;
    } else if (year == "normal") {
        cout << playInTotal << endl;
    }

	return 0;
}

File: Python/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P09_Volleyball.py
import math
year = input()
p = int(input())  # празници в годината
h = int(input())  # брой уикенди на село

weekends_Sofia = (48 - h) * 3/4
play_in_holidays = p * 2/3
total_play = weekends_Sofia + play_in_holidays + h

total_visokosna = total_play * 1.15

if year == "normal":
    print(math.floor(total_play))
else:
    print(math.floor(total_visokosna))


